dotnet/src/Connectors/Connectors.Memory.Weaviate/WeaviateMemoryStore.cs

```diff
@@ -48,6 +49,11 @@ public class WeaviateMemoryStore : IMemoryStore
         DefaultIgnoreCondition = JsonIgnoreCondition.WhenWritingNull
     };
```

Comment:
As @dmytrostruk well pointed out.

```suggestion
        Converters = { new ReadOnlyMemoryConverter() }
    };
```